### PR TITLE
fix: make entire operation-tag collapsible, rename some functions (TDX-1887)

### PR DESCRIPTION
**Pull request by davidma415**
This PR fixes the issue with the `OperationTag`  not focusing on the right elements. It now focuses on the entire header element, and toggles on keypress / click. Also renames some functions in other files.

@Mierenga I added the `aria-expanded` attribute on the containing `div` instead, as I think that makes more sense semantically (since the div is being collapsed / expanded, not the header).

### CODE REVIEW

File: src/components/OperationTag.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    let tagDescription = tagObj.getIn(["tagDetails", "description"], null)
+    let tagExternalDocsDescription = tagObj.getIn(["tagDetails", "externalDocs", "description"])
+    let tagExternalDocsUrl = tagObj.getIn(["tagDetails", "externalDocs", "url"])
+
+    let isShownKey = ["operations-tag", tag]
+    let showTag = layoutSelectors.isShown(isShownKey, docExpansion === "full" || docExpansion === "list")
+
+    return (
+      <div
```
Comment (tomaszwylezek) [inline src/components/OperationTag.js:45]: Any reason why we changed from `section` to `div`?

**Review by tomaszwylezek — COMMENTED** [review]
One queston about html semantic element change

**Review by tomaszwylezek — APPROVED** [review]
lgtm

File: src/components/OperationTag.js (hunk 1 above)
Comment (davidma415) [reply src/components/OperationTag.js:45]: Accident - let me change it back.

**Pull request merged**